doc/handling-high-traffic.md

```diff
@@ -0,0 +1,29 @@
+# Handling High Traffic
+
+## Introduction
+Starting in 2025 Wikibase cloud started occasionally receiving a large amount of traffic.
```

Comment:
Is it also the type of traffic as well as the amount? I.e. querying pages that aren't cached and accept query parameters (e.g. recent changes).